### ISSUE TITLE
Menu hitTest() crash when outside Menu area

### ISSUE THREAD

**Issue by GreyEcologist** [CosmicMind/Material#672]
Since version 2.4.2, when the menu is opened and enabled, if user press outside the menu area the hitTest crashes. I'm not able to see the changes from 2.4.1 to 2.4.2 but my issue started since that version.

This is my current set up for the Menu:

```
        let btn1: FabButton = FabButton()
        btn1.depthPreset = .depth2
        btn1.pulseAnimation = .centerWithBacking
        btn1.backgroundColor = buttonColor
        btn1.tintColor = Color.white
        btn1.setImage(R.image.ic_add_black()!, for: .disabled)
        btn1.image = R.image.ic_add_white()!
        btn1.addTarget(self, action: #selector(self.handleMenu(open:)), for: .touchUpInside)
        
        var image = R.image.facebooklogo()!
        let btn2: FabButton = FabButton()
        btn2.tag = 1
        btn2.contentMode = .scaleToFill
        btn2.depthPreset = .depth1
        btn2.pulseColor = Color.white
        btn2.backgroundColor = Color.indigo.darken1
        btn2.borderColor = Color.white
        btn2.borderWidth = 2
        btn2.image = image
        btn2.addTarget(self, action: #selector(self.socialButton(button:)), for: .touchUpInside)
        
        image = R.image.twitterlogo()!
        let btn3: FabButton = FabButton()
        btn3.tag = 2
        btn3.contentMode = .scaleToFill
        btn3.depthPreset = .depth1
        btn3.backgroundColor = Color.lightBlue.base
        btn3.pulseColor = Color.white
        btn3.borderColor = Color.white
        btn3.borderWidth = 2
        btn3.image = image
        btn3.addTarget(self, action: #selector(self.socialButton(button:)), for: .touchUpInside)
        
        menu.direction = .up
        menu.delegate = self
        menu.views = [btn1, btn2, btn3]
```

The exact error is a exc_bad_access at line  112 of the Menu class inside the hitTest() method:
```
let p = v.convert(point, from: self)
```

**Comment by danieldahan** [comment]
Hey, what version are you using now? I believe I have fixed this, and if it has not been released yet, it is definitely coming up in the next release. I spent a lot of time fixing up this control for Material 2.5. Can you confirm the latest has it? Material 2.5 is near ready and will have a definite fix. Here is an article you can read that describes the new FABMenu's usage: [Transitions With FABMenuController](http://www.cosmicmind.com/danieldahan/transitions-with-fabmenucontroller).

**Comment by GreyEcologist** [comment]
@danieldahan Thanks, will have a look. I've tried with 2.4.11, 2.4.5 and 2.4.2, same result: I have to regress to 2.4.1 for it to work.

**Comment by danieldahan** [comment]
Okay, there is a fix in Material 2.5 for this. So when I release it, I will close this issue and you can update :)

**Comment by danieldahan** [comment]
Hey, I have tried both the [CardTableView](https://github.com/CosmicMind/Samples/tree/master/Graph/CardTableView) and [MenuController](https://github.com/CosmicMind/Samples/tree/master/Material/Programmatic/MenuController) samples with [Material 2.4.12](https://github.com/CosmicMind/Material/releases/tag/2.4.12) and cannot reproduce the issue. Material 2.4.12 is a new release today. Please give it a try, and if the issue persists, reopen the issue. Also, if you could send me a reproducible sample to support@cosmicmind.com that would be great. Thank you :)
